### ISSUE TITLE
Tests not executed in CI anymore

### ISSUE THREAD

**Issue by echoix** [microsoft/XamlBehaviorsWpf#120]
**Describe the bug**
With the newest activity on the repo today (good sign!) I observed that the tests aren't being ran in CI as checks.

The tests seem to be run as part of an Azure Pipeline, but the last time a commit included this check was in https://github.com/microsoft/XamlBehaviorsWpf/commit/9db3c14f65e51798e3bace1c8d96b5547628621e (November 21, 2022)

No test steps seem to be configured in the [main.yml](https://github.com/microsoft/XamlBehaviorsWpf/blob/b259d249a4a48fff44d99499888d48d315adb99a/.github/workflows/main.yml) workflow. 

**To Reproduce**
1. Go to https://github.com/microsoft/XamlBehaviorsWpf/commits/master
2. Click on a green check or red x after a commit to look at the checks status for a commit. Choose one on April 10, 2023, and one on November 21, 2022.
3. Scroll down to look for a Azure Pipelines check.
4. See that the commits on April 10, 2023 don't include an Azure Pipelines check. 

**Expected behavior**
Either work on enabling back the Azure Pipelines (but external contributors can't monitor it without having appropriate login credentials), or modernize the GitHub Actions workflow to build and test at the same time.

**Screenshots**

Commit https://github.com/microsoft/XamlBehaviorsWpf/commit/7678354c86bfbe36f0b6e909175ca279419790dc doesn't have tests ran. 
![image](https://user-images.githubusercontent.com/27212526/231032447-d51298f6-6bf5-46da-8281-108a0a911323.png)
![image](https://user-images.githubusercontent.com/27212526/231032437-8688367b-7e8a-46ae-905b-b8dfb675c482.png)

**Desktop (please complete the following information):**
 - Version with bug: 7678354c86bfbe36f0b6e909175ca279419790dc and later
 - Last known working version: 9db3c14f65e51798e3bace1c8d96b5547628621e 

**Reproduction Link**

**Comment by brianlagunas** [comment]
Wow, great catch. Unfortunately I don’t have access to any of the DevOps infrastructure. Hopefully @mgoertz-msft can look into this for us. 

I do have another PR in the works so it would be nice to get our tests running with CI again.

**Comment by echoix** [comment]
I was surprised to have a email notification of an update on the repo, I had enabled them since I was using it and had some usability bugs/irritations when using it with VS2022, but the repo has been very quiet for a long time.


But either way, if you aren't experienced with Actions, there is always this to help out: https://docs.github.com/en/actions/migrating-to-github-actions/automated-migrations/automating-migration-with-github-actions-importer,
but there might be some better patterns with dotnet projects now than a port of a three years old Azure Pipeline.

**Comment by brianlagunas** [comment]
Actually, I didn’t realize this repo was using GitHub actions. I was never made aware of that change. I’m familiar with GitHub actions and should be able to help out there too.

**Comment by brianlagunas** [comment]
It’s confusing because I see Azure Pipelines AND GitHub Actions in this repo.

**Comment by echoix** [comment]
> It’s confusing because I see Azure Pipelines AND GitHub Actions in this repo. 

But it's not a problem, you can always use the best tool for a job, and if it means using Azure Pipelines and GitHub Actions, and any other, it isn't a problem, and each can run independently.

**Comment by brianlagunas** [comment]
Yes, publishing a release should definitely be a manual process. For now, we can just add the unit tests to the GitHub Actions. This will help ensure a breaking change is less likely to be introduced via a PR. However, ideally, the release build would also run unit tests.

**Comment by brianlagunas** [comment]
Tests have been added to the GitHub Actions
